src/mapdata.h

```diff
@@ -429,6 +429,7 @@ extern ter_id t_null,
        t_vat,
        t_rootcellar,
        t_cvdbody, t_cvdmachine,
+       t_nanofab, t_nanofab_body,
```

Comment:
These two variables aren't needed. The *very* few places that use those ids can use the string directly (e.g. `ter_str_id( "t_nanofab_body" )` instead of `t_nanofab_body`). The variable `t_nanofab` is not used anywhere at all.

Reply:
Thanks for the tip